internal/lox_files_test.go

```
@@ -0,0 +1,126 @@
+package internal
+
+import (
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"testing"
+)
+
+func TestEmptyTrailingLines(t *testing.T) {
+	os.Chdir("../test_programs")
```

golangci-lint (1.64.8)

Error return value of `os.Chdir` is not checked

(errcheck)